### ISSUE TITLE
Connectivity browser - make the component work again with the latest version of the data

### ISSUE THREAD

**Issue by ddelpiano** [VirtualFlyBrain/geppetto-vfb#987]
- Uncomment component from the VFBMain and the menu.
- Update the component with the cypher query provided here https://app.zenhub.com/workspaces/vfb15-59c292c4286ea849ed3c9326/issues/virtualflybrain/geppetto-vfb/657 (follow up the thread for the latest)
- Once the component is resumed please report here what changes are required to make this work again with the new data pushed by Robb and David (so that we can better estimate and scheduled all the required cards).

- Please also uncomment the term info logic to add instances to this component and test if this works fine
- if the term info logic does not work open a new card to fix that but keep it out from the scope of this card.

**Comment by jrmartin** [comment]
The component has been updated to work using updated query on https://app.zenhub.com/workspaces/vfb15-59c292c4286ea849ed3c9326/issues/virtualflybrain/geppetto-vfb/658

It works using instances : 'VFB_jrchjrch','VFB_jrchjsfu', 'VFB_jrchjs1t'
![image.png](https://images.zenhubusercontent.com/5b8f86ce39827c509079f60a/4691bdc8-25e1-4cd8-90cd-fead3f16f243)

Tested again queries FBbt_00002303 and FBbt_00002442, giving back no graph.

Things to do:
1) Term Info links for Connectivity Browser are not showing up. An issue has been opened for this : https://github.com/VirtualFlyBrain/geppetto-vfb/issues/1007

2) Circuit Browser crashing when closing and re-opening from Tools bar. https://github.com/VirtualFlyBrain/geppetto-vfb/issues/1008

**Comment by dosumis** [comment]
> Should new query work with FBbt_00002303 and FBbt_00002442? @Robbie1977 @dosumis

TLDR:  No

Longer answer: 
New query only works where edges have a weight - which right now is just Individuals.   In discussion with @Robbie1977 we decided that that the MVP should be for connectivity queries for Individuals only as we hit performance issues with the generalised version.  We can easily limit inputs to this by using labels (gross types) to limit SOLR autocomplete and links from TermInfo.

**Comment by dosumis** [comment]
@jrmartin - can you share a high res screenshot?  Would be helpful to see layout.

**Comment by dosumis** [comment]
Some notes here for future discussion:

* A better edge display would be 2 arrows (one for each direction), each annotated with weight.
* Layout - checking pinning.  Is it possible to pin (one? all?) non-root search terms to right?

**Comment by jrmartin** [comment]
> @jrmartin - can you share a high res screenshot? Would be helpful to see layout.

@dosumis Here are three high res screenshots:
With hops set to 2:
![Screenshot from 2021-02-17 10-42-29](https://user-images.githubusercontent.com/4562825/108252289-55077e00-710d-11eb-97a6-bf4729c1d884.png)

With hops set to 3:
![Screenshot from 2021-02-17 10-42-45](https://user-images.githubusercontent.com/4562825/108252261-4de07000-710d-11eb-891e-87d365e6af2d.png)

With hops set to 4:
![Screenshot from 2021-02-17 10-44-10](https://user-images.githubusercontent.com/4562825/108252253-4b7e1600-710d-11eb-835b-ff34fd970c30.png)

**Comment by dosumis** [comment]
Thanks for images.  Looks like key functionality is not in place.  The first query term is root and should be pinned to the left (just as in context browser, it is pinned to the bottom).  Without this, column based layout makes not sense.

**Comment by ddelpiano** [comment]
@dosumis this is what we are going to work in the next sprint and we were still working on defining the requirements across the last 2 sprints (and I will narrow that down this week), functionalities are not expected to work yet, @jrmartin here just restored the component to work with the new data.
Thx
